assets/js/components/notifications/SubtleNotification.js

```diff
@@ -47,76 +46,70 @@ function SubtleNotification( {
 	Icon,
 	ctaLink,
 	ctaLabel,
+	className,
 	onCTAClick,
 	isCTALinkExternal,
+	isDismissible = true,
 	dismissLabel = __( 'Ok, got it', 'google-site-kit' ),
-	onDismiss,
+	onDismiss = () => {},
```

Comment:
On further thought, I don't think this default value is required as well.

```suggestion
	onDismiss,
```